src/xmipp/bindings/matlab/xmipp_nma_read_alignment.cpp

```diff
@@ -73,12 +73,12 @@ void mexFunction( int nlhs, mxArray *plhs[],
   std::vector<double> lambda;
   FOR_ALL_OBJECTS_IN_METADATA(mdImages)
```

Comment:
replace `FOR_ALL...` by `for (size_t objId : mdImages.ids())` (multiple occurrences)